addons/website/views/snippets/s_three_columns.xml

```diff
@@ -43,10 +37,4 @@
     </section>
 </template>
 
-<template id="assets_snippet_s_three_columns_css_000" inherit_id="website.assets_frontend">
-    <xpath expr="//link[last()]" position="after">
-        <link rel="stylesheet" type="text/scss" href="/website/static/src/snippets/s_three_columns/000.scss"/>
-    </xpath>
-</template>
```

Comment:
Should I also create an empty 001.scss, for clarity (otherwise there might be a jump from 000 to 002 in a future version) or maybe use a special data-vcss tag (`data-vcss="no_css"`), or should I just bump the vcss number?